ports-wip/syncthing/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash -x
+
+# This code Copyright 2012 Todd Mortimer <todd.mortimer@gmail.com>
+#
+# You may do whatever you like with this code, provided the above
+# copyright notice and this paragraph are preserved.
+
+
+set -e
+source ../../lib.sh
+DISTVER="syncthing"
+DISTSUFFIX="git"
+GIT_URL="https://github.com/syncthing/syncthing.git"
```

Comment:
@berryamin from what I've read golang is not happening on QNX in the foreseeable future so this can go :(